includes/Database_Upgrader.php

```diff
@@ -51,6 +51,34 @@ class Database_Upgrader extends Service_Base implements Activateable {
 	 */
 	const PREVIOUS_OPTION = 'web_stories_previous_db_version';
 
+	/**
+	 * Array of classes to run migration routines.
+	 *
+	 * @var array
+	 */
+	const ROUTINES = [
+		'1.0.0' => Migrations\Update_1::class,
+		'2.0.0' => Migrations\Replace_Conic_Style_Presets::class,
+		'2.0.1' => Migrations\V2_Add_Term::class,
+		'2.0.2' => Migrations\Remove_Broken_Text_Styles::class,
+		'2.0.3' => Migrations\Unify_Color_Presets::class,
+		'2.0.4' => Migrations\Update_Publisher_Logos::class,
+		'3.0.0' => Migrations\Add_Stories_Caps::class,
+		'3.0.1' => Migrations\Rewrite_Flush::class,
+		'3.0.2' => Migrations\Rewrite_Flush::class,
+		'3.0.3' => Migrations\Yoast_Reindex_Stories::class,
+		'3.0.4' => Migrations\Add_Poster_Generation_Media_Source::class,
+		'3.0.5' => Migrations\Remove_Unneeded_Attachment_Meta::class,
+		'3.0.6' => Migrations\V3_Add_Term::class,
+	];
```

Comment:
I think now would be an excellent time to re-think our naming for all the migrations here.

For example, I don't like the two outliers with `V2_` / `V3_` prefixes.

We could rename them to `Add_Media_Source_Editor` and `Add_Media_Source_Video_Optimization`